ui/src/app/base/components/NodeName/NodeNameFields/NodeNameFields.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
   saving?: boolean;
 };
 
-export const MachineNameFields = ({ saving }: Props): JSX.Element => {
+export const NodeNameFields = ({ saving }: Props): JSX.Element => {
```

Comment:
Do you think we could use the `DomainSelect` component here?

Reply:
Yep, good idea!